pages/content/amp-dev/documentation/guides-and-tutorials/integrate/web-player-guide.md

```diff
@@ -0,0 +1,387 @@
+---
+$title: Web Story Player - User’s guide
+description: 'The Web Story Player is the best way for you to give users a rich user experience on your sites using Web Stories.'
+formats:
+  - websites
+  - stories
+---
+
+# Web Story Player - User’s guide
+
+The Web Player has significantly matured (thanks for all the feedback and insights), and is in a great place for publishers, developers, tools and platforms to start using in their site experiences. This update recaps key capabilities from the Web Story Player, and provides guides on common implementation patterns.
+
+
+## What is the Web Player?
+
+{{ image('/static/img/docs/web-player-main.gif', 750, 625,   align='center', caption='Web Story Player demo' alt='web player demo' ) }}
+
+The Web Story Player is the best way for you to give users a rich user experience on your sites using Web Stories.  Think of it like a video player: having a single video in your device is fine, but put it in a video player live on the web, and it unlocks a number of features and benefits: There’s additional controls (e.g. captions), it lives among millions of other videos that can provide related content at the end of your original video – which adds discoverability, and more.
+
```

Comment:
Inset note here about non-AMP vs AMP and direct readers to the embedding WST in AMP pages doc

Reply:
Added it below under a `[tip type="note"]`